app/browser/api/ledgerNotifications.js

```diff
@@ -26,23 +27,31 @@ const text = {
   paymentDone: undefined,
   addFunds: locale.translation('addFundsNotification'),
   tryPayments: locale.translation('notificationTryPayments'),
-  reconciliation: locale.translation('reconciliationNotification')
+  reconciliation: locale.translation('reconciliationNotification'),
+  backupKeys: locale.translation('backupKeys')
 }
 
-const pollingInterval = 15 * ledgerUtil.milliseconds.minute // 15 * minutes
+const pollingInterval = process.env.LEDGER_NOTIFICATION ? ledgerUtil.milliseconds.second : 15 * ledgerUtil.milliseconds.minute // 15 * minute default production
 let intervalTimeout
 const displayOptions = {
   style: 'greetingStyle',
   persist: false
 }
 const nextAddFundsTime = 3 * ledgerUtil.milliseconds.day
+let backupNotifyInterval = process.env.LEDGER_NOTIFICATION ? [10 * ledgerUtil.milliseconds.second, 15 * ledgerUtil.milliseconds.second] : [7 * ledgerUtil.milliseconds.day, 14 * ledgerUtil.milliseconds.day]
```

Comment:
`10 * ledgerUtil.milliseconds.second, 15 * ledgerUtil.milliseconds.second` -> `2 * ledgerUtil.milliseconds.minute, 5 * ledgerUtil.milliseconds.minute`. It's hard to QA it if things are in seconds